app/models/package.rb

```diff
@@ -82,7 +82,6 @@ def self.search(search_text, item_id, options = {})
           state: state
         )
       end
-    records = records.where(received_quantity: 1) unless options[:show_quantity_items]
```

Comment:
Why we removed it ?

Reply:
To turn on ability to view quantity items